### ISSUE TITLE
Some chrome tasks still running on background after quit() method called

### ISSUE THREAD

**Issue by hornyja4** [SeleniumHQ/selenium#9718]
## 🐛 Bug Report

Hi, I tried to migrate our project to Selenium 4 (beta-4) and run all our scenarios. I faced the issue with still running chrome tasks (4-5 from 20) on background even if `quit()` method was called and browser windows were closed. This issue happens when i'm running tests in parallel. 

Here is a log with some exceptions from Selenium itself. The exception is **_Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem_** which means **An existing connection was forcibly closed by the remote host**

These tasks takes a lot of CPU.

![image](https://user-images.githubusercontent.com/15084479/128237172-5751d82b-7ab6-4291-b252-4c3b0d8a50c7.png)

```
20:15:14.910 [AsyncHttpClient-1-4] . TRACE WebSocket08FrameEncoder - Encoding WebSocket Frame opCode=8 length=26
20:15:14.910 [AsyncHttpClient-1-4] . TRACE LoggingHandler - [id: 0xcee2ef77, L:/127.0.0.1:64035 - R:localhost/127.0.0.1:65124] WRITE: 32B
         +-------------------------------------------------+
         |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
+--------+-------------------------------------------------+----------------+
|00000000| 88 9a 0c 92 4e ce 0f 7a 19 ab 6e d6 3c a7 7a f7 |....N..z..n.<.z.|
|00000010| 3c ee 6f fe 21 bd 65 fc 29 ee 7f fd 2d a5 69 e6 |<.o.!.e.)...-.i.|
+--------+-------------------------------------------------+----------------+
20:15:14.910 [Driver Command Executor] . DEBUG NettyRequestSender - Using pooled Channel '[id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264]' for 'DELETE' to 'http://localhost:38264/session/654168405d12651e9a3e04d37cfe2eeb'
20:15:14.940 [AsyncHttpClient-1-4] . TRACE LoggingHandler - [id: 0xcee2ef77, L:/127.0.0.1:64035 - R:localhost/127.0.0.1:65124] FLUSH
20:15:14.940 [Driver Command Executor] . DEBUG NettyRequestSender - Using open Channel [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] for DELETE '/session/654168405d12651e9a3e04d37cfe2eeb'
20:15:14.940 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] WRITE: 194B
         +-------------------------------------------------+
         |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
+--------+-------------------------------------------------+----------------+
|00000000| 44 45 4c 45 54 45 20 2f 73 65 73 73 69 6f 6e 2f |DELETE /session/|
|00000010| 36 35 34 31 36 38 34 30 35 64 31 32 36 35 31 65 |654168405d12651e|
|00000020| 39 61 33 65 30 34 64 33 37 63 66 65 32 65 65 62 |9a3e04d37cfe2eeb|
|00000030| 20 48 54 54 50 2f 31 2e 31 0d 0a 55 73 65 72 2d | HTTP/1.1..User-|
|00000040| 41 67 65 6e 74 3a 20 73 65 6c 65 6e 69 75 6d 2f |Agent: selenium/|
|00000050| 34 2e 30 2e 30 2d 62 65 74 61 2d 34 20 28 6a 61 |4.0.0-beta-4 (ja|
|00000060| 76 61 20 77 69 6e 64 6f 77 73 29 0d 0a 43 6f 6e |va windows)..Con|
|00000070| 74 65 6e 74 2d 54 79 70 65 3a 20 61 70 70 6c 69 |tent-Type: appli|
|00000080| 63 61 74 69 6f 6e 2f 6a 73 6f 6e 3b 20 63 68 61 |cation/json; cha|
|00000090| 72 73 65 74 3d 75 74 66 2d 38 0d 0a 68 6f 73 74 |rset=utf-8..host|
|000000a0| 3a 20 6c 6f 63 61 6c 68 6f 73 74 3a 33 38 32 36 |: localhost:3826|
|000000b0| 34 0d 0a 61 63 63 65 70 74 3a 20 2a 2f 2a 0d 0a |4..accept: */*..|
|000000c0| 0d 0a                                           |..              |
+--------+-------------------------------------------------+----------------+
20:15:14.940 [AsyncHttpClient-1-4] . TRACE LoggingHandler - [id: 0xcee2ef77, L:/127.0.0.1:64035 - R:localhost/127.0.0.1:65124] READ COMPLETE
20:15:14.940 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] FLUSH
20:15:14.940 [AsyncHttpClient-1-4] . TRACE LoggingHandler - [id: 0xcee2ef77, L:/127.0.0.1:64035 ! R:localhost/127.0.0.1:65124] INACTIVE
20:15:14.940 [AsyncHttpClient-1-4] . DEBUG WebSocketHandler - Channel Closed: [id: 0xcee2ef77, L:/127.0.0.1:64035 ! R:localhost/127.0.0.1:65124] with attribute NettyResponseFuture{currentRetry=5,
	isDone=1,
	isCancelled=0,
	asyncHandler=org.asynchttpclient.ws.WebSocketUpgradeHandler@6046bda3,
	nettyRequest=org.asynchttpclient.netty.request.NettyRequest@45c4333c,
	future=java.util.concurrent.CompletableFuture@21cbcce0[Completed normally],
	uri=ws://localhost:65124/devtools/browser/42b6e264-592b-4340-8a7d-034916579a39,
	keepAlive=true,
	redirectCount=0,
	timeoutsHolder=null,
	inAuth=0,
	touch=1628100906525}
20:15:14.967 [AsyncHttpClient-1-4] . TRACE WebSocketHandler - Connection was closed abnormally (that is, with no close frame being received).
20:15:14.967 [AsyncHttpClient-1-4] . DEBUG ChannelManager - Closing Channel [id: 0xcee2ef77, L:/127.0.0.1:64035 ! R:localhost/127.0.0.1:65124] 
20:15:14.967 [AsyncHttpClient-1-4] . TRACE LoggingHandler - [id: 0xcee2ef77, L:/127.0.0.1:64035 ! R:localhost/127.0.0.1:65124] UNREGISTERED
20:15:15.037 [AsyncHttpClient-1-3] . TRACE LoggingHandler - [id: 0xe718199b, L:/127.0.0.1:64034 - R:localhost/127.0.0.1:65124] READ COMPLETE
20:15:15.037 [AsyncHttpClient-1-3] . TRACE LoggingHandler - [id: 0xe718199b, L:/127.0.0.1:64034 - R:localhost/127.0.0.1:65124] EXCEPTION: java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:233) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:223) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:15:15.038 [AsyncHttpClient-1-3] . DEBUG HttpHandler - Unexpected I/O exception on channel [id: 0xe718199b, L:/127.0.0.1:64034 - R:localhost/127.0.0.1:65124]
java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:233) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:223) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:15:15.038 [AsyncHttpClient-1-3] . DEBUG ChannelManager - Closing Channel [id: 0xe718199b, L:/127.0.0.1:64034 - R:localhost/127.0.0.1:65124] 
20:15:15.038 [AsyncHttpClient-1-3] . TRACE LoggingHandler - [id: 0xe718199b, L:/127.0.0.1:64034 - R:localhost/127.0.0.1:65124] CLOSE
20:15:15.038 [AsyncHttpClient-1-3] . TRACE LoggingHandler - [id: 0xe718199b, L:/127.0.0.1:64034 ! R:localhost/127.0.0.1:65124] USER_EVENT: io.netty.channel.socket.ChannelInputShutdownReadComplete@6aaf8d63
20:15:15.038 [AsyncHttpClient-1-3] . TRACE LoggingHandler - [id: 0xe718199b, L:/127.0.0.1:64034 ! R:localhost/127.0.0.1:65124] INACTIVE
20:15:15.038 [AsyncHttpClient-1-3] . DEBUG HttpHandler - Channel Closed: [id: 0xe718199b, L:/127.0.0.1:64034 ! R:localhost/127.0.0.1:65124] with attribute DISCARD
20:15:15.038 [AsyncHttpClient-1-3] . TRACE LoggingHandler - [id: 0xe718199b, L:/127.0.0.1:64034 ! R:localhost/127.0.0.1:65124] UNREGISTERED
20:15:15.070 [AsyncHttpClient-1-1] . DEBUG DefaultChannelPool - Entry count for : http://localhost:65124 : 1
20:15:15.070 [AsyncHttpClient-1-1] . DEBUG DefaultChannelPool - Entry count for : http://localhost:38264 : 1
20:15:15.070 [AsyncHttpClient-1-1] . DEBUG DefaultChannelPool - Adding Candidate expired Channel [id: 0xe718199b, L:/127.0.0.1:64034 ! R:localhost/127.0.0.1:65124] isIdleTimeoutExpired=false isRemotelyClosed=true isTtlExpired=false
20:15:15.070 [AsyncHttpClient-1-1] . DEBUG DefaultChannelPool - Closing Idle Channel [id: 0xe718199b, L:/127.0.0.1:64034 ! R:localhost/127.0.0.1:65124]
20:15:15.070 [AsyncHttpClient-1-1] . DEBUG DefaultChannelPool - Closed 1 connections out of 2 in 0 ms
20:15:15.083 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] READ: 122B
         +-------------------------------------------------+
         |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
+--------+-------------------------------------------------+----------------+
|00000000| 48 54 54 50 2f 31 2e 31 20 32 30 30 20 4f 4b 0d |HTTP/1.1 200 OK.|
|00000010| 0a 43 6f 6e 74 65 6e 74 2d 4c 65 6e 67 74 68 3a |.Content-Length:|
|00000020| 31 34 0d 0a 43 6f 6e 74 65 6e 74 2d 54 79 70 65 |14..Content-Type|
|00000030| 3a 61 70 70 6c 69 63 61 74 69 6f 6e 2f 6a 73 6f |:application/jso|
|00000040| 6e 3b 20 63 68 61 72 73 65 74 3d 75 74 66 2d 38 |n; charset=utf-8|
|00000050| 0d 0a 63 61 63 68 65 2d 63 6f 6e 74 72 6f 6c 3a |..cache-control:|
|00000060| 6e 6f 2d 63 61 63 68 65 0d 0a 0d 0a 7b 22 76 61 |no-cache....{"va|
|00000070| 6c 75 65 22 3a 6e 75 6c 6c 7d                   |lue":null}      |
+--------+-------------------------------------------------+----------------+
20:15:15.083 [AsyncHttpClient-1-5] . DEBUG HttpHandler - 

Request DefaultFullHttpRequest(decodeResult: success, version: HTTP/1.1, content: EmptyByteBufBE)
DELETE /session/654168405d12651e9a3e04d37cfe2eeb HTTP/1.1
User-Agent: selenium/4.0.0-beta-4 (java windows)
Content-Type: application/json; charset=utf-8
host: localhost:38264
accept: */*

Response DefaultHttpResponse(decodeResult: success, version: HTTP/1.1)
HTTP/1.1 200 OK
Content-Type: application/json; charset=utf-8
cache-control: no-cache
content-length: 14

20:15:15.083 [AsyncHttpClient-1-5] . DEBUG ChannelManager - Adding key: http://localhost:38264 for channel [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264]
20:15:15.084 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] READ COMPLETE
20:15:16.091 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] READ COMPLETE
20:15:16.091 [AsyncHttpClient-1-17] . TRACE LoggingHandler - [id: 0x4c0579e5, L:/127.0.0.1:65123 - R:localhost/127.0.0.1:38264] READ COMPLETE
20:15:16.092 [AsyncHttpClient-1-17] . TRACE LoggingHandler - [id: 0x4c0579e5, L:/127.0.0.1:65123 - R:localhost/127.0.0.1:38264] EXCEPTION: java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:233) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:223) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:15:16.092 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] EXCEPTION: java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:233) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:223) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:15:16.093 [AsyncHttpClient-1-17] . DEBUG HttpHandler - Unexpected I/O exception on channel [id: 0x4c0579e5, L:/127.0.0.1:65123 - R:localhost/127.0.0.1:38264]
java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:233) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:223) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:15:16.093 [AsyncHttpClient-1-17] . DEBUG ChannelManager - Closing Channel [id: 0x4c0579e5, L:/127.0.0.1:65123 - R:localhost/127.0.0.1:38264] 
20:15:16.093 [AsyncHttpClient-1-5] . DEBUG HttpHandler - Unexpected I/O exception on channel [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264]
java.io.IOException: Stávající připojení bylo vynuceně ukončeno vzdáleným hostitelem
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:233) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:223) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.63.Final.jar:4.1.63.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
20:15:16.093 [AsyncHttpClient-1-17] . TRACE LoggingHandler - [id: 0x4c0579e5, L:/127.0.0.1:65123 - R:localhost/127.0.0.1:38264] CLOSE
20:15:16.093 [AsyncHttpClient-1-5] . DEBUG ChannelManager - Closing Channel [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] 
20:15:16.093 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 - R:localhost/127.0.0.1:38264] CLOSE
20:15:16.093 [AsyncHttpClient-1-17] . TRACE LoggingHandler - [id: 0x4c0579e5, L:/127.0.0.1:65123 ! R:localhost/127.0.0.1:38264] USER_EVENT: io.netty.channel.socket.ChannelInputShutdownReadComplete@6aaf8d63
20:15:16.093 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 ! R:localhost/127.0.0.1:38264] USER_EVENT: io.netty.channel.socket.ChannelInputShutdownReadComplete@6aaf8d63
20:15:16.093 [AsyncHttpClient-1-17] . TRACE LoggingHandler - [id: 0x4c0579e5, L:/127.0.0.1:65123 ! R:localhost/127.0.0.1:38264] INACTIVE
20:15:16.093 [AsyncHttpClient-1-17] . DEBUG HttpHandler - Channel Closed: [id: 0x4c0579e5, L:/127.0.0.1:65123 ! R:localhost/127.0.0.1:38264] with attribute DISCARD
20:15:16.093 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 ! R:localhost/127.0.0.1:38264] INACTIVE
20:15:16.093 [AsyncHttpClient-1-17] . TRACE LoggingHandler - [id: 0x4c0579e5, L:/127.0.0.1:65123 ! R:localhost/127.0.0.1:38264] UNREGISTERED
20:15:16.093 [AsyncHttpClient-1-5] . DEBUG HttpHandler - Channel Closed: [id: 0x29e8007f, L:/127.0.0.1:61604 ! R:localhost/127.0.0.1:38264] with attribute DISCARD
20:15:16.093 [AsyncHttpClient-1-5] . TRACE LoggingHandler - [id: 0x29e8007f, L:/127.0.0.1:61604 ! R:localhost/127.0.0.1:38264] UNREGISTERED
```

## To Reproduce

Run multiple tests in parallel and call `quit()` after tests

## Expected behavior

All chrome tasks are closed together with browser windows.

## Environment

OS: Windows 10
Browser: Chrome
Browser version: 92
Browser Driver version: ChromeDriver 92.0.4515.43
Language Bindings version: Java: 4.0.0-beta-4

**Comment by diemol** [comment]
We had a leak in Java that was leaving this processes open, it was fixed a couple of weeks ago and it should be released in 1-2 weeks.
So, this is a duplicate of https://github.com/SeleniumHQ/selenium/issues/9666
